File: 25.12.25/palindrome_number.java
class Solution {
    public boolean isPalindrome(int x) {
        int same;
        same=x;
        int reverse=0;
        while(x>0)
        {
            reverse=(reverse*10)+x%10;
            x=x/10;
        }
        if(reverse==same)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

File: 01.01.26/palindrome_number.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        cleaned=""
        for ch in s:
            if ch.isalnum():
                cleaned +=ch.lower()
            reversed_string=cleaned[::-1]
        if cleaned==reversed_string:
               return True
        else:
               return False
